expiringdict/__init__.py

```diff
@@ -110,7 +113,7 @@ def get(self, key, default=None, with_age=False):
     def items(self):
         """ Return a copy of the dictionary's list of (key, value) pairs. """
         r = []
-        for key in self:
+        for key in list(self.keys()):
```

Comment:
That makes sense. I will check version first.
In python 3.5, there are some implementation changes of OrderedDict, so directly iterating will give RuntimeError: OrderedDict mutated during iteration.